.circleci/config.yml

```diff
@@ -9,334 +9,330 @@ orbs:
 executors:
   docker-executor:
     docker:
-      - image: circleci/openjdk:17-buster-node-browsers-legacy
+    - image: circleci/openjdk:17-buster-node-browsers-legacy
 jobs:
   build:
     docker:
-      - image: cimg/openjdk:17.0.0-node
-      - image: docker:17.05.0-ce-git
+    - image: cimg/openjdk:17.0.0-node
```

Comment:
please limit your changes to only the lines that need changes,
Lot of formating changes